mso/datasource_mso_schema_site_vrf_route_leak.go

```diff
@@ -0,0 +1,93 @@
+package mso
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/mso-go-client/client"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func dataSourceMSOSchemaSiteVrfRouteLeak() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceMSOSchemaSiteVrfRouteLeakRead,
+
+		SchemaVersion: version,
+
+		Schema: (map[string]*schema.Schema{
+			"schema_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"template_name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"site_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"vrf_name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"target_vrf_schema_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"target_vrf_template_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"target_vrf_name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"tenant_name": &schema.Schema{
```

Comment:
Shouldn't this be Required?

Reply:
no it is not required for the data source, only for the configuration in the resource

Reply:
ok